### PR TITLE
[#2349] improved(scripts): Add scripts directory for jdbc backend ddl scripts

### PR DESCRIPTION
**Pull request by xloya**
### What changes were proposed in this pull request?

Add a separate scripts directory to save the ddl sql statements that jdbc backend needs to execute.
77d1a79a-faf8-4622-981a-74c3b50478db

### Why are the changes needed?

Fix: #2349

### CODE REVIEW

**Comment by xloya** [comment]
@jerryshao Could you take a look of this? Thanks.

**Comment by xloya** [comment]
@yuqi1129 Please also take a look of this, thanks.

File: core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ private void init(String type, Config config) {
 
   private void prepareJdbcTable() {
     // Read the ddl sql to create table
-    String scriptPath = "h2/h2-init.sql";
+    String scriptPath = "relational-entity-store/jdbc/h2/schema-0.5.0-h2.sql";
```
Comment (yuqi1129) [inline core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java:113]: What's the meaning of `0.5.0`?

Comment (xloya) [reply core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java:113]: This refers to the next release version. We may add some ddl statements to the new version in the future. Users can execute initialization statements according to the access version. Referring to the implementation in Hive Metastore:
image

**Comment by yuqi1129** [comment]
Additionally, it appears that we lack a mechanism to verify whether the configurations of relational entity store is well-configurated at the start.  Only when we are about to operate metalakes, catalogs and so on can we confirm that everything is well. Is there a way to verify it?

**Comment by xloya** [comment]
> Additionally, it appears that we lack a mechanism to verify whether the configurations of relational entity store is well-configurated at the start. Only when we are about to operate metalakes, catalogs and so on can we confirm that everything is well. Is there a good way to verify it?

A simple way is that when we start the MySQL backend, we try to perform operations similar to `select * from xxx limit 1;` for each table. This can ensure that the database connection is normal and the corresponding table is created.

File: core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java (hunk 1 above)
Comment (yuqi1129) [reply core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java:113]: Got it.

**Comment by xloya** [comment]
@jerryshao Could you take a look of this? Thanks.

**Comment by jerryshao** [comment]
Maybe we don't need to have too many folders for script, "scripts/mysql/xxx" is enough, WDYT?

**Comment by xloya** [comment]
> Maybe we don't need to have too many folders for script, "scripts/mysql/xxx" is enough, WDYT?

Sure, I don't have much preference for this.

**Comment by jerryshao** [comment]
> > Maybe we don't need to have too many folders for script, "scripts/mysql/xxx" is enough, WDYT?
> 
> Sure, I don't have much preference for this.

Then let's do this.

**Review by jerryshao — APPROVED** [review]
LGTM, thanks @xloya for your contribution.

**Comment by yuqi1129** [comment]
The current CI has some issues due to an image update by Github. Please rebase the PR after #2784 is merged @xloya

**Comment by jerryshao** [comment]
Finally, pass the CI, merging to main, thanks @xloya for your work.

**Pull request merged**